app/exceptions.py

```diff
@@ -1,3 +1,6 @@
+"""Exception classes for configuration-related errors."""
```

Comment:
Future exceptions might not necessarily be configuration-related, right?

Reply:
you can decide. I am only starting with this, but we can decide if we don't want to enforce at the module/file level.